module/LearnZF2Authentication/src/LearnZF2Authentication/Controller/IndexController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+
+namespace LearnZF2Authentication\Controller;
+
+use Zend\Mvc\Controller\AbstractActionController;
+use Zend\View\Model\ViewModel;
+use Zend\Mvc\MvcEvent;
+
+/**
+ * Class IndexController.
+ *
+ * @author Stanimir Dimitrov <s.dimitrov@weblunatix.com>
+ */
+class IndexController extends AbstractActionController
+{
+    /**
+     * @var ViewModel
+     */
+    public $view = null;
+
+    public function __construct()
+    {
+        $this->view = new ViewModel();
+    }
+
+    /**
+     * @param MvcEvent $e
+     */
+    public function onDispatch(MvcEvent $e)
+    {
+        parent::onDispatch($e);
+    }
```

Comment:
yeah, it's my way of calling the ViewModel(). Thanks for the coverage, was wondering where to looks for more info.